doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
+            <td><code>WD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Undergoing pre-FCD review (or) undergoing comment resolution from pre-FCD review (or) undergoing comment resolution from PCD period</td>
+            <td><code>CD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Pre-FCD review complete (or) ready for FCD ballot (or) undergoing FCD ballot (or) ready for PCD (or) currently in PCD period</td>
+            <td><code>CD</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot complete, undergoing comment resolution</td>
+            <td><code>FCD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot comment resolution complete (or) undergoing pre-DP review (or) ready for DP ballot (or) undergoing DP ballot </td>
+            <td><code>FCD</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>FCD ballot complete with no comments (or) ready for ST Audit (or) undergoing ST audit</td>
+            <td><code>DP</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>ST audit complete (or) waiting for publication</td>
+            <td><code>PUB</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Published</td>
+            <td><code>PUB</code></td>
+            <td><code>pub</code></td>
+          </tr>
+        </tbody>
+      </table>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing these statuses. 
+      </p>
+
+    </section>
+
+    <section id="sec-document-github-releases">
+      <h3>GitHub Releases</h3>
+      <p>
+        Whenever the status of the document dictates as per <a href="#tab-document-mapping-status"></a>, that a merged commit to the main branch that updates the document to <code>&lt;meta itemprop="pubState" content="pub" /&gt;</code>, a release shall be created in the GitHub repo. 
+      </p>
+      <p>
+        The GitHub release shall be named <code>rYYYY-pubState</code>, with a tag assigned to the release that matches the release name. If creating a CD, itterate the <code>pubState</code>, starting with <code>1</code>. 
+      </p>
+      <p class="example">
+        <code>r2023-fcd</code> or <code>r2023-cd2</code>
+      </p>
+      <p class="note">
+        The itteration of CD is to account for multiple possible PCD periods. 
+      </p>
+      <p>
+        When creating the next update on the repo, update <code>latestEditionTag</code> in the <code>.smpte-build.json</code> file to the value equal to the tag name of this latest release. This will trigger the tooling to create a link for <code>Redline to most recent edition</code> during any PRs and merges to the main branch. 
```

Comment:
What is a "next update"? Do you mean "every time pubStage" advances?

Reply:
Yea, was looking for better wording, will address.

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96